src/components/controller/controller.ts

```diff
@@ -0,0 +1,91 @@
+import ICar from './ICar';
+import IWinner from './IWinner';
+
+class Controller {
+    cars: ICar[];
+    winners: IWinner[];
+    baseURL: string;
+    constructor() {
+        this.baseURL = 'http://localhost:3000';
+        this.cars = [];
+        this.winners = [];
+    }
+
+    getCars(callback: (data: ICar[]) => void) {
```

Comment:
You can simply return fetched cars from this method, passing cb to process fetched data is overcompicated